examples/kiva/simple2.py

```diff
@@ -11,22 +11,23 @@ def simple():
     gc.set_fill_color((0, 0, 0))
     gc.rect(99, 100, 300, 300)
     gc.draw_path()
-    gc.save(tempfile.mktemp(suffix=".bmp"))
+
+    with tempfile.NamedTemporaryFile(suffix='.bmp') as fid:
+        gc.save(fid.name)
```

Comment:
I'm a little curious if this is necessary at all or just left over from debugging/experimentation. Perhaps it's needed for a  side effect on the `bmp_array` attribute?

Reply:
I'm not seeing anything supporting why it would be needed, and the example seems to run the same with or without it

Reply:
If the behavior is unchanged without it, I would remove it. It's not useful to have noise like this in example code.